<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文本编辑器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="02_Editor.qrc">
    <normaloff>:/images/AppIcon.ico</normaloff>:/images/AppIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="m_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="m_actNew"/>
    <addaction name="m_actOpen"/>
    <addaction name="separator"/>
    <addaction name="m_actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="m_actCut"/>
    <addaction name="m_actCopy"/>
    <addaction name="m_actPaste"/>
    <addaction name="separator"/>
    <addaction name="m_actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="m_actBold"/>
    <addaction name="m_actItalic"/>
    <addaction name="m_actUnderline"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="m_actToolBar"/>
    <addaction name="m_actStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="m_actAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actNew"/>
   <addaction name="m_actOpen"/>
   <addaction name="m_actExit"/>
   <addaction name="separator"/>
   <addaction name="m_actCut"/>
   <addaction name="m_actCopy"/>
   <addaction name="m_actPaste"/>
   <addaction name="m_actClear"/>
   <addaction name="separator"/>
   <addaction name="m_actBold"/>
   <addaction name="m_actItalic"/>
   <addaction name="m_actUnderline"/>
  </widget>
  <action name="m_actNew">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/new.bmp</normaloff>:/images/new.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_actOpen">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/open.bmp</normaloff>:/images/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_actExit">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="m_actCut">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/cut.bmp</normaloff>:/images/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="m_actCopy">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/copy.bmp</normaloff>:/images/copy.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="m_actPaste">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/paste.bmp</normaloff>:/images/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="m_actClear">
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="m_actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/bld.bmp</normaloff>:/images/bld.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="m_actItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/itl.bmp</normaloff>:/images/itl.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="m_actUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="02_Editor.qrc">
     <normaloff>:/images/under.bmp</normaloff>:/images/under.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="m_actToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
  </action>
  <action name="m_actStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
  </action>
  <action name="m_actAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="02_Editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_actExit</sender>
   <signal>triggered()</signal>
   <receiver>EditorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actCut</sender>
   <signal>triggered()</signal>
   <receiver>m_edit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actCopy</sender>
   <signal>triggered()</signal>
   <receiver>m_edit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actPaste</sender>
   <signal>triggered()</signal>
   <receiver>m_edit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actClear</sender>
   <signal>triggered()</signal>
   <receiver>m_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
